ait/core/test/test_tlm.py

```diff
@@ -118,6 +119,85 @@ def test_fld_defn_notitle(self):
         os.remove(test_yaml_test2)
 
 
+class TestDerivationDefinition(object):
+    test_yaml_deriv1 = '/tmp/test_deriv1.yaml'
+
+    yaml_docs_deriv1 = (
+        '- !Packet\n'
+        '  name: OCO3_1553_EHS\n'
```

Comment:
Let's make a follow up ticket and make some changes to the names for consistency